### ISSUE TITLE
Use .innerText instead of .innerHTML 

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#3457]
See: https://cheatsheetseries.owasp.org/cheatsheets/AJAX_Security_Cheat_Sheet.html

The use of .innerText will prevent most XSS problems as it will automatically encode the text.

**Comment by kevinreiss** [comment]
is this for stackmap?

**Comment by christinach** [comment]
No, I came across this information https://cheatsheetseries.owasp.org/cheatsheets/AJAX_Security_Cheat_Sheet.html

**Comment by christinach** [comment]
@kevinreiss I updated the ticket description.

**Comment by kevinreiss** [comment]
We use .innerHTML [23 times](https://github.com/search?q=repo%3Apulibrary%2Forangelight%20.innerHTML&type=code) at the moment.
